src/material/datepicker/datepicker.md

```diff
@@ -1,6 +1,24 @@
 The datepicker allows users to enter a date either through text input, or by choosing a date from
 the calendar. It is made up of several components and directives that work together.
 
+To start using the datepicker you will need to import the datepicker module and set the datepicker adapter to use:
+
+```ts
+import { NgModule } from '@angular/core';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatNativeDateModule } from '@angular/material/core';
+
+@NgModule({
+  imports: [
+    // ...
+    MatDatepickerModule,
+    MatNativeDateModule,
```

Comment:
I think that if we list both adapters out like this, people might get confused and import both of them. Also we'd have to remember to update the list if we decide to add new adapters.

Reply:
perhaps this section can move so it's closer to the overview? https://github.com/angular/components/blob/ea31bca2c186fd496dd3daf2e692617c060c64d7/src/material/datepicker/datepicker.md#choosing-a-date-implementation-and-date-format-settings

also there's already a spot that lists the providers available: https://github.com/angular/components/blob/9eeb4b5aa0473c0815a73bfe3c2ed6164d86a3b3/src/material/datepicker/datepicker-errors.ts#L13

I think it's okay to list just the native date module since it is included with `@angular/material` when it's installed and because it'll always be around whereas other providers could become available. This would avoid any confusion with more than one provider being imported.